docs/sdk/core/examples/app2_test.go

```diff
@@ -0,0 +1,83 @@
+package app
+
+import (
+	"testing"
+	"github.com/tendermint/tendermint/crypto"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/stretchr/testify/require"
+)
+
+// Test encoding of app2Tx is correct with both msg types
+func TestEncoding(t *testing.T) {
+	// Create privkeys and addresses
+	priv1 := crypto.GenPrivKeyEd25519()
+	priv2 := crypto.GenPrivKeyEd25519()
+	addr1 := priv1.PubKey().Address().Bytes()
```

Comment:
do we need the .Bytes() here ?

Reply:
Yea, sdk.AccAddress expects a byte slice not common.HexBytes